frame/staking/src/benchmarking.rs

```diff
@@ -637,8 +944,19 @@ benchmarks! {
 	}
 
 	chill_other {
-		let (_, controller) = create_stash_controller::<T>(USER_SEED, 100, Default::default())?;
-		Staking::<T>::validate(RawOrigin::Signed(controller.clone()).into(), ValidatorPrefs::default())?;
+		// Clean up any existing state.
+		clear_validators_and_nominators::<T>();
+
+		// A worst case scenario includes the voter being a bag head, so we can reuse the rebag
```

Comment:
note: removal of non-terminal is 2 storage access (prev and next), removal of terminal is also 2 (prev OR next, and the bag).